war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -616,7 +616,13 @@ function makeButton(e,onclick) {
     if(h!=null)
         btn.addListener("click",h);
     var be = btn.get("element");
-    Element.addClassName(be,clsName);
+    var classesSeparatedByWhitespace = clsName.split(' ');
+    for (var i = 0; i < classesSeparatedByWhitespace.length; i++) {
+        var singleClass = classesSeparatedByWhitespace[i];
+        if (singleClass) {
+            be.classList.add(singleClass);
+        }
+    }
```

Comment:
Could this be replaced by something like `clsName.split(' ').forEach(function (className) { if (className) be.classList.add(className) })` ?